scripts/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env php
+<?php
+echo "Running unit tests..." . PHP_EOL;
+exec('./vendor/bin/phpunit test', $output, $returnCode);
```

Comment:
i'm not a fan of adding such pre-commit hooks, we should rather set this up on CI level

Reply:
Yeah, CI makes more sense 🤔 But we don't have CI for DFP Query Tool. We wanted to add some kind of a protection to this repo as soon as possible because last week we almost merged changes that were breaking unit tests 😊